swagger.json

```diff
@@ -6257,6 +6474,117 @@
+            "description": "User properties. Only one of organizationRole or tagRoles is required.",
+            "required": true,
+            "in": "body",
+            "schema": {
+                "type": "object",
+                "required": [
+                    "name",
+                    "email",
+                    "authType"
+                ],
+                "properties": {
+                    "name" : {
+                        "type": "string",
+                        "description": "The first and last name of the user.",
+                        "example": "Jane Doe"
+                    },
+                    "email": {
+                        "type": "string",
+                        "description": "The email address of this user.",
+                        "example": "user@company.com"
+                    },
+                    "organizationRole" : {
+                        "type": "string",
+                        "description": "The name of the role the user is assigned to for the organization. Leave this blank to give only tag access. This must match one of the preset roles or the name of a custom role in the organization. This value is case insensitive.",
+                        "example": "Read-Only Admin"
+                    },
+                    "tagRoles" : {
+                        "type": "array",
+                        "description": "If specified, the user will be a tag administrator for these tags. If left blank, the user has access to the entire organization.",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "tagName": {
+                                    "type": "string",
+                                    "description": "The name of the tag to grant this user access to.",
+                                    "example": "North Carolina Distribution"
+                                },
+                                "role" : {
+                                    "type": "string",
+                                    "description": "The name of the role the user is assigned to for this tag. This must match one of the preset roles or the name of a custom role in the organization. This value is case insensitive.",
+                                    "example": "Read-Only Admin"
+                                }
+                            }
+                        }
+                    },
+                    "authType": {
+                        "type": "string",
+                        "enum": [
+                            "basic",
+                            "saml"
+                        ],
+                        "description": "The authentication type the user uses to authenticate. To use SAML your organization must have a configured SAML integration."
+                    }
+                }
+            }
+        },
+        "updateUserParam": {
```

Comment:
Nit: Can `updateUserParam` and `createUserParam` share the same base object, using `allOf`, to avoid copy/pasting these fields?

Reply:
I wasn't aware of `allOf`, yes that makes sense!